inc/class-shortcode-ui.php

```diff
@@ -75,13 +65,6 @@ public function register_shortcode_ui( $shortcode_tag, $args = array() ) {
 			array_splice( $args['attrs'], $index, 1 );
 		}
 
-		// strip invalid
-		foreach ( $args as $key => $value ) {
-			if ( ! array_key_exists( $key, $defaults ) ) {
-				unset( $args[ $key ] );
-			}
-		}
-
```

Comment:
Probably could be tackled as a separate issue. But we're setting the defaults in the models themselves. I don't think we should replicate this in PHP as well.